### ISSUE TITLE
NTR: Lamellated/'basket-woven' thickened glomerular basement membranes (level 6)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6279]
Lamellated/'basket-woven' thickened glomerular basement membranes (level 6)
parent:Thickened glomerular basement membranes with 'basket woven' appearance (NEW TERM)
def:Prominent GBM reflecting an diffuse and relativly uniform increase in thickness (subjective estimate) with splitting and fragmentation of the lamina densa into several strands, forming a “basket-weave” pattern @avizrosenberg @yongqunh

**Comment by pnrobinson** [comment]
@avizrosenberg @yongqunh

Is this a synonym of this existing term?
https://hpo.jax.org/app/browse/term/HP:0030034

**Comment by avizrosenberg** [comment]
essentially a synonym, only the HPO term includes diffuse.

**Comment by pnrobinson** [comment]
I think the existing HPO term does not need to have "diffuse" in its label, since we never discriminated between focal and diffuse with this term. I will make this a synoynm and if later we need to discriminate, we can add new children terms.
